src/PointwiseFunctions/AnalyticData/GrMhd/SphericalTorus.cpp

```diff
@@ -135,8 +167,10 @@ SphericalTorus::inv_jacobian(const tnsr::I<T, 3>& source_coords) const {
 
   auto& sin_theta = get<1, 2>(inv_jacobian);
   auto& cos_theta = get<0, 2>(inv_jacobian);
+  // intermediate variable for equatorial compression
+  get<2, 2>(inv_jacobian) = cubic_compression(get<1>(source_coords));
```

Comment:
Again.  Go through and find all cases of this.

Reply:
done.